src/objects.js

```diff
@@ -1,41 +1,54 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name,
+    age,
+  }
```

Comment:
Small formatting point:
```suggestion
  return { name, age };
```